src/GovUk.Education.ExploreEducationStatistics.Admin/Security/AuthorizationHandlers/ViewSpecificTopicAuthorizationHandler.cs

```diff
@@ -1,58 +1,15 @@
-
-using System.Linq;
-using System.Threading.Tasks;
-using GovUk.Education.ExploreEducationStatistics.Common.Services.Security;
-using GovUk.Education.ExploreEducationStatistics.Content.Model;
-using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
 using Microsoft.AspNetCore.Authorization;
-using Microsoft.EntityFrameworkCore;
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.Security.AuthorizationHandlers
 {
     public class ViewSpecificTopicRequirement : IAuthorizationRequirement
```

Comment:
I forgot that we can actually check a claim without a handler at all by making a policy require a claim rather than have any requirements. Just this on it's own would remove the need for this class but I think we can go further because the Themes and Topics are only accessed under the taxonomy management section. I've just applied that taxonomy management permission to everything.

This removes the handler for themes and topics, all the requirements, and the claim for all topics.

The handler for taxonomy management is replaced by making the policy dependent on the claim too.